README.md

```diff
@@ -1,25 +1,78 @@
+The growth rate of stars and growth rate of commits are the metrics we will use to decide which repositories to invest in. The final product also needs to have the functionality to conduct Ad hoc custom analysis. We can better assess investment potential with specific, tailored analyses.
 
-## Data Source
+# Data Source
 The choice for the data source was between the Github API and GH Archive.
 
-The pros of the Github API are that it provides live fresh data and no 3rd party is involved. The con of Github API is that there is complex load logic; we would end up exceeding the alotted number of API calls. The pros of GH archive is that it has simple load logic; it is easier for us to gather all the necessary data. The cons of GH archive are that it refreshes every hour (is not the most up-to-date data) and a 3rd party is involved.
+### Github API
+#### Pros:
+* Live, fresh data
+* No 3rd party involved
+  
+#### Cons:
+* Complex load logic; we would exceed the alotted number of API calls
 
-Ultimately, we chose our data source to be GH Archive due to the feasibility issue that the Github API introduces. As the client only requires a daily refresh on the data, the refresh frequency of 1 hour is acceptable. The potential issues that could arise due to the fact that the data is from a 3rd party can be mitigated with data quality tests.
+### GH Archive
+#### Pros:
+*  Simple load logic; it is easier for us to gather all the necessary data
 
-## Data Format
-The client requests that the data be delivered in the format of a SQL database.
+#### Cons:
+*  Refreshes every hour (not the most up-to-date data)
+*  3rd party is involved
 
-## Update Frequency
-The client requests that the data be updated daily.
+Ultimately, we chose our data source to be GH Archive due to the feasibility issue that the Github API introduces. As we only need the data to be updated daily, the refresh frequency of 1 hour is acceptable. The potential issues that could arise due to the fact that the data is from a 3rd party can be mitigated with data quality tests.
+
+# Models
+
+For this project we will be using the Kimball Model. It is a framework to designing data structures which construct several star schemas to fulfill various reporting needs. The Kimball Model can be broken down into three aspects:
+* Facts: the measurements that result from a business process event (almost always numeric)
+* Dimensions: the information that provides who, what, where, when, why, and how context
+* The grain: establishes what a single fact table row represents
+
+### Facts and Dimensions Models
+#### Facts Fields
+* Stars
+  * event_id
+  * repo_id
+  * user_id
+  * event_date
+* Commits
+  * event_id
+  * repo_id
+  * user_id
+  * event_date
+
+#### Dimensions Fields
+* Repositories
```

Comment:
Here the naming convention is `dim_` for dimension tables, so `dim_repositories` etc.